### ISSUE TITLE
Image metadata overlaps timestamp

### ISSUE THREAD

**Issue by robintown** [vector-im/element-web#22310]
### Steps to reproduce

1. Use the default image size setting
2. Use bubble layout
3. Send or receive a portrait image
4. Hover over it

#### What did you expect?

The timestamp should be readable

#### What happened instead?

It's overlapped by the metadata plinth:

![overlap](https://user-images.githubusercontent.com/48614497/169697022-08baddda-6de9-47c4-b6a5-4d09b9116e0b.png)

### Operating system

NixOS unstable

### Browser information

Firefox 100.0

### URL for webapp

develop.element.io

### Application version

Element version: 6c7f6639838d-react-11cb48176e01-js-25afb7cb3cc6 Olm version: 3.2.8

### Homeserver

Synapse 1.58.0

### Will you send logs?

No

**Comment by t3chguy** [comment]
Related https://github.com/vector-im/element-web/issues/21651
